default/stringtables/en.txt

```diff
@@ -19870,15 +19967,18 @@ Mira
 Mirach
 Mirak
 Mirzam
+Mithuna
 Mizar
 Mothallah
+Mrigasheerisham
```

Comment:
a bit long...

Reply:
was there before, just reordered alphabetically

Reply:
is the 11-letter-limit (which i read somewhere) hard and binding? indeed, name seems long enough to split in two

Reply:
I don't know of any particular limit being document anywhere. I just thought that name looks quite a bit longer, and could be awkward in the UI.